packages/shared/src/contexts/OnboardingContext.tsx

```diff
@@ -82,7 +87,10 @@ export const OnboardingContextProvider = ({
   };
   const { sidebarRendered } = useSidebarRendered();
   const showArticleOnboarding =
-    sidebarRendered && alerts?.filter && onboardingV2 === OnboardingV2.Control;
+    sidebarRendered &&
+    alerts?.filter &&
+    (onboardingV2 === OnboardingV2.Control ||
+      onboardingV3 === OnboardingV3.Control);
```

Comment:
What about here? @idoshamun

Reply:
I think it's not needed here as well. V3 should just indicate if we should show the new version (v3) or control (v2). Should not impact the rest

Reply:
Removed.